gui/MainWindow.cxx

```diff
@@ -2744,11 +2741,11 @@ void MainWindow::nextSlide()
 }
 
 //-----------------------------------------------------------------------------
-void MainWindow::setActiveCamera(int id)
+void MainWindow::setActiveFrame(kwiver::vital::frame_id_t id)
 {
   QTE_D();
 
-  int lastFrameId = d->frames.isEmpty() ? 1 : d->frames.lastKey();
+  auto const lastFrameId = d->frames.isEmpty() ? 0 : d->frames.lastKey();
```

Comment:
Is changing this to from `1` to `0` going to generate a bunch of warnings now when there are no frames loaded?  I think the idea was that the active frame is allowed to be `1` when nothing is loaded so that it won't complain.  If `0` the conditional below will always send invalid frame messages regardless of what frame number is set.

Reply:
Does that actually happen? AFAICT this should never be called with no video loaded. It's only called by selecting a frame via the UI (which is disabled when no video is loaded, and I don't see a way to unload video?) or by a running tool (in a manner that we would expect the frame to be valid).

Reply:
Yeah, you could be right that this code will not be called if the video is not loaded.  However, I seem to remember sometime in the past where this was being called without a video loaded or maybe just as a video was loaded and the frame were still empty.  Regardless, I think that `1` was intentional to avoid warnings invalid frames.  My memory of this could be entirely obsolete now.

Reply:
I have not, in any case, observed the warning to be emitted in practice, although I have no tested extensively. (I have not, at least, seen it just from starting the app or loading imagery. Additionally, it's unclear what `MainWindowPrivate::setActiveCamera` will do when given an invalid frame, which could theoretically happen with the old logic.)